.eslintrc

```diff
@@ -6,6 +6,7 @@
     "eslint-plugin-amphtml-internal",
     "eslint-plugin-google-camelcase",
     "jsdoc",
+    "notice",
```

Comment:
There's scope for a minor cleanup here, if you're game.

I believe the prefix `eslint-plugin-` prefix is optional. Remove the prefix from `amphtml-internal` and `google-camelcase` (and make sure `gulp lint` still works).

Reply:
Always happy to cleanup, done.